docs/source/concepts/cl_columns.rst

```diff
@@ -21,37 +28,99 @@ The Cleanlab columns are listed below in the table. Detailed explanations of eac
+
 .. _suggested_label:
 ``suggested_label``
 ---------------
-Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue == False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
+Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue`` marked as ``False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
 
 Outliers
 ========
 
 .. _is_outlier:
 ``is_outlier``
 -------
-Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
\ No newline at end of file
+Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
+
+Note: for projects on multi-label tabular datasets, outliers are currently not computed, and ``is_outlier`` will always be False. 
+=======
+
+.. _outlier_score:
+``outlier_score``
+-----------
+Contains a score bounded between 0 and 1, which is used to determine whether a sample is an outlier. The lower the score of a sample, the more likely it is to be an outlier.
+
+Ambiguous
+=========
+
+.. _is_ambiguous:
+``is_ambiguous``
+----------
+Contains a boolean value, with ``True`` indicating that the sample is likely to be ambiguous. Ambiguous samples are those that do not obviously belong to a single class.
+
+.. _ambiguous_score:
+``ambiguous_score``
+-------------
+Contains a score bounded between 0 and 1, which is used to determine whether a sample is ambiguous. The lower the score of a sample, the more likely it is to be ambiguous.
+
+High Confidence
+===============
+
+.. _is_high_confidence_given_label:
+``is_high_confidence_given_label``
+---------------
+Contains a boolean value, with ``True`` indicating that the given label of the sample is likely to be correct, so the sample can be safely used in downstream tasks.
+
+Near Duplicates
+===============
+
+.. _is_near_duplicate:
+``is_near_duplicate``
+----------------
+Contains a boolean value, with ``True`` indicating that the sample is likely to be a near duplicate of another sample. Near duplicates are two or more examples in a dataset that are extremely similar (or identical) to each other, relative to the rest of the dataset.
+
+.. _near_duplicate_score:
+``near_duplicate_score``
+------------------
+Contains a score bounded between 0 and 1, which is used to determine whether a sample is a near duplicate. The lower the score of a sample, the more likely it is to be a near duplicate of another sample.
+
+.. _near_duplicate_id:
+``near_duplicate_id``
+----------------
+Contains an integer ID for each sample, where samples with the same ID are near duplicates of each other. The IDs range from 0 upwards. Samples that do not have near duplicates are assigned an ID of -1.
```

Comment:
Remember to update this text once we switch the representation to use NA instead of -1.